documentation/TESTING_PLAN.md

```diff
@@ -16,7 +16,16 @@ In general, we're approaching testing bottom-up, with each unit function tested
 
 ### Unit-level tests
 
-- TBD 10
+- Test that impassible walls are impassible by the player.
+- Test that the level is complete/congratulatory message when and only when all boxes are moved to designated goal cells.
+- Test when a level is complete, the user is prompted to move to the next level or jump to a different level.
+- Test that all boxes are pushable by the player.
+- Test the score is tracking player movement.
+- Test that the undo button undoes moves.
+- Test the reset button reset’s the board properly.
```

Comment:
Same as above, the backend doesn't understand what a button is, so we can move this to integration testing.